templates/reset_body.html

```diff
@@ -106,17 +106,21 @@
       font-size: 28px !important;
       line-height: 36px !important;
       letter-spacing: -0.01em !important;
-      color: #3D3C40 !important;
+      color: #3F4350 !important;
     }
 
     .subTitle div {
       font-size: 16px !important;
       line-height: 24px !important;
-      color: rgba(61, 60, 64, 0.64) !important;
+      color: rgba(63, 67, 80, 1) !important;
```

Comment:
when opacity value is 100% (1), the alpha channel is not needed, so no need to use the four channels rgba function. The rgb(r, g, b) is enough.